lib/ElementHandle.js

```diff
@@ -149,6 +156,18 @@ class ElementHandle extends JSHandle {
     return {x, y, width, height};
   }
 
+  /**
+   * @return {!Promise<?object>}
+   */
+  async boxModel() {
+    const result = await this._getBoxModel();
+
+    if (!result)
```

Comment:
nit: to my taste, i'd replace lines 164-168 with a one-liner:

```js
return result ? result.model : null;
```

Reply:
this is not relevant after my last change (moving to {x, y} representation)